docs/wxm-devices/D1-bundle/D1-troubleshooting.mdx

```diff
@@ -0,0 +1,125 @@
+---
+id: D1-troubleshooting
+title: WeatherXM D1 bundle (WB1200) Troubleshooting
+sidebar_label: D1 Bundle Troubleshooting
+slug: /wxm-devices/D1-bundle/D1-troubleshooting
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+## Weather Station
+
+### Weather station not reporting data to WG1200 Gateway. There are dashes (--) on the display console.
+
+First make sure you WG1200 Gateway frequency matches your weather station (see [Change frequency](/wxm-devices/D1-bundle/D1-connect-and-claim#change-frequency-optional))
+
+Check the LED on the weather station (see [Reset Button and LED Indicator](/wxm-devices/D1-bundle/D1-assemble#outdoor-weather-station)) for flashing.
+The LED should flash every 16 seconds.
+
+If the LED is not flashing every 16 seconds, replace the batteries in the weather station.
+If the batteries were recently replaced, check the polarity.
+
+If the above does not solve the issue, there may be a temporary loss of communication due to reception loss related to interference or other location factors, or the D1 Gateway has not been reset.
+The solution may be as simple as powering down and up the WG1200 Gateway. Make sure you use lithium non-rechargeable batteries, especially in case of cold weather environments.
+
+1. With the weather station array and WG1200 Gateway 3m/10ft away from each other,
+   remove the batteries from the weather station and wait 10 seconds. Put the batteries back in.
+1. WG1200 Gateway should display all the observations measured by the weather station.
+1. Try to place the WG1200 Gateway at the desirable point and check that it receives data.
+   Ensure the power adapter if used is not too close to the D1Gateway and the D1 Gateway is not close to other electrical noise generating devices
+   such as TVs, monitors, computers and transmitting devices.
+1. If you still have intermittent problems, move the weather station and the WG1200 Gateway closer together,
+   but not closer than 1.5m/5ft or further than 50m/164ft.
+   Also check that there are no metal barriers like aluminum siding
+   or metal wall framing along the line of sight between the weather station and WG1200 Gateway.
+   Relocate the weather station and WG1200 Gateway as necessary to avoid obstacles.
+   Depending on natural barriers you may also have to move the weather station higher and/or closer.
+
+### Temperature looks higher than expected.
+
+If the temperature is slightly higher (e.g., +2 - 5°C/35-41°F) than expected,
+then assure that the weather station is within a significant distance from heat sources
+like ventilation systems and active chimneys.
+Also make sure that the weather station is mounted on a 2m/6.5ft (or more) metallic pole and there are no obstacles close by reading [deployments examples for Proper Station Installation](/deployment-examples).
```

Comment:
by reading [deployments examples for Proper Station Installation](/deployment-examples). ---> by (see [Best Practices for Proper Station Installation](/wxm-devices/D1-bundle/install-weather-station#best-practices-for-proper-station-installation)).